index.md

```diff
@@ -56,20 +56,18 @@ __Technical information__
 
 ## Project and community
 
-Charmed HPC is a project of the [Ubuntu High-Performance Computing
+Charmed HPC is an open source project of the [Ubuntu High-Performance Computing
 community](https://ubuntu.com/community/governance/teams/hpc).
 Interested in contributing bug fixes, patches, documentation, or feedback?
 Want to join the Ubuntu HPC community? You've come to the right place!
 
 Here's some links to help you get started with joining the community:
 
-<!-- TODO: Add page with contributing guidelines. -->
-<!-- TODO: Add page with more information on how to get support for Charmed HPC. -->
-
-* [Ubuntu Code of Conduct](https://ubuntu.com/community/ethos/code-of-conduct)
-* [Join the conversation on Matrix](https://matrix.to/#/#hpc:ubuntu.com)
+* [Read and follow the Ubuntu Code of Conduct](https://ubuntu.com/community/ethos/code-of-conduct)
+* [Join the Ubuntu HPC community on Matrix](https://matrix.to/#/#hpc:ubuntu.com)
 * [Get the latest news on Discourse](https://discourse.ubuntu.com/c/hpc/151)
-* [Ask and answer questions on GitHub](https://github.com/orgs/charmed-hpc/discussions/categories/q-a)
+* [Visit the Charmed HPC GitHub Organization](https://github.com/charmed-hpc)
+* [Ask for support](https://github.com/orgs/charmed-hpc/discussions/categories/support)
```

Comment:
[nit]: I want to encourage users to try and help each rather than just have people "screaming into the void" asking for help, and we should indicate that our support forum is on GitHub within the link text rather than just have a generic "Ask for support" link that doesn't indicate where the final destination is.

```suggestion
* [Ask and answer support questions on GitHub](https://github.com/orgs/charmed-hpc/discussions/categories/support)
```